Update REVOKE information to say that in cypher 25 you get errors for impossible revoke commands (#2009)
neo4j/docs-operations: modules/ROOT/pages/authentication-authorization/database-administration.adoc

====

Common errors, such as misspellings or attempts to revoke privileges that have not been granted or denied, will lead to notifications.

> In Cypher 25, notifications for impossible `REVOKE` commands, where a user, a role, or a database does not exist, have been replaced with errors.

Reviewer: do we have any examples of the Cypher 5 behaviour that we should now prepend with `CYPHER 5` and add notes on that they would fail in Cypher 25? Or is these 3 sentences the only thing on this topic that we had? 🤔

Author: I can not find anything more on this topic than those 3 sentences. Looks like all the examples we have are successful ones, not examples that will fail.

> Notifications for impossible `REVOKE` commands where any of the user, role or database does not exist have been replaced with errors in Cypher 25.

Reviewer 2: ```suggestion
In Cypher 25, notifications for impossible `REVOKE` commands, where a user, a role, or a database does not exist, have been replaced with errors.
```
